### PR TITLE
no free code if you take on investments

### PR DESCRIPTION
**Pull request by dirjk**
simple proposal for a front end website. currently it does nothing except display the wallstreetsbets logo, but the full tech stack is there ready to go for whatever designs and functionalities you want.

you can run it locally or do a full build and deploy it to any webserver. instructions are in the README.

### CODE REVIEW

**Comment by dirjk** [comment]
also just want to mention that my job is literally being a principle front end engineer for one of the biggest companies in the USA, so if you need someone to help manage the site and get organized I can help with that too.

**Comment by jcsumlin** [comment]
I for one love React and would like to contribute to working on this!

File: ReactFrontEnd/src/build_entries/index.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+import mainLogo from '../../assets/cover1.jpg'
+import styles from './index.css'
```
Comment (jcsumlin) [inline ReactFrontEnd/src/build_entries/index.jsx]: Could be wrong but this looks like an un-used import statement. 
Should just be `import './index.css'`

**Review by jcsumlin — APPROVED** [review]
It works!
Very minimalistic but for a front end this will do for now.
My inline comment appears to be more of a coding style instance than a broken code instance. This can be merged as is.

File: ReactFrontEnd/src/build_entries/index.jsx (hunk 1 above)
Comment (dirjk) [reply ReactFrontEnd/src/build_entries/index.jsx]: yeah, probably. i whipped it together as a proof of concept in something like 10 minutes. there's no linting standards installed yet either ha ha

**Comment by DrewMcArthur** [comment]
i wonder if this repo might end up being more of an organizing space, with subrepos that contain more specific functionality like this.  

i’m also a huge react fan, so definitely support this! just not sure what exactly this project will be - might be easier to get this merged in if all the code you’re adding gets tossed in an enclosing folder (or sub repo, though i haven’t done much w them to know how those work)

File: ReactFrontEnd/package.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        "sass": "^1.32.6",
+        "sass-loader": "^11.0.1",
+        "style-loader": "^2.0.0",
+        "url-loader": "^4.1.1",
+        "webpack": "^5.11.1",
+        "webpack-cli": "^4.3.1",
+        "webpack-dev-server": "^3.11.2"
+    },
+    "dependencies": {
+        "css-loader": "^5.0.2"
```
Comment (cooperhurt) [inline ReactFrontEnd/package.json]: I vote for scss here instead of css.

**Comment by jcsumlin** [comment]
@DrewMcArthur 
> i wonder if this repo might end up being more of an organizing space, with subrepos that contain more specific functionality like this.

I think this is a possibility for sure but projects that contain both the front and back end can exist with the right deployment instructions

File: ReactFrontEnd/package.json (hunk 2 above)
Comment (dirjk) [reply ReactFrontEnd/package.json]: so if you look in the webpack.config.js file on lines 32 - 40, this merge request is actually works with all 3: scss, css, and sass files. there is an issue with the different loaders on windows machines in webpack 5 and sass, and this repo has the fix installed as a fall back as the 3rd loader (sass-loader).

as for the quick demo, i just used a plain old .css file for initial pull request purposes but if you'd prefer to use scss or sass we can definitely enforce that standard going forward.

edit for clarity: this pull request already works with scss we would just have to decide on which tech to use going forward.

Comment (foosiee) [reply ReactFrontEnd/package.json]: You get my vote as well, also before we go off merging this project is certainly going to consist of multiple repos so better naming convention for this repo might be good.

**Comment by hackerncoder** [comment]
> i wonder if this repo might end up being more of an organizing space, with subrepos that contain more specific functionality like this.

I think creating an orginization (not just user) and multiple projects might be the way forward.

**Comment by hackerncoder** [comment]
Someone has already taken the name WallStreetBetsAnalytics. Just to ensure it didn't get snatched I have created WSBAnalytics.

**Comment by foosiee** [comment]
Also should come up with a fun name for this sure it will come in due time

File: ReactFrontEnd/README.md [hunk 3]
```diff
@@ -1,32 +1,4 @@
...
-
-# Anything else I should know?
-
-Yes! This repo is set up to use the most recent version of React, React v17. Additionally it uses webpack 5 and babel for its production build. This is literally bleeding edge front end technology here! 
-
-# but what about IE 11?
-
-first, shut up. We hate IE 11 and if you use it you should upgrade. But with that said, yes, we support IE 11 but only so much as babel supports it. we don't support it with styles or more advanced visualization tools. Its not the 1800s anymore, you need to upgrade.
\ No newline at end of file
+don't. I saw that you are talking to angel investors and I have no intention of working for free for your bullshit startup. If you take on investment and want my input, you will have to pay my normal hourly rates which you probably can't afford.
```
Comment (MantasSmilin) [inline ReactFrontEnd/README.md:4]: Full disclosure - I don't know what happened. Just sharing information posted on discord.

> Auzy 2/19 at 12:25 AM EST
@everyone After our conversation with zjz and seeing his response on reddit, we believe there may have been some miscommunication. We ARE NOT planning on selling out to some "angel investor" or anything. There has been one conversation that has occurred between us and said investor, and we are currently unsure what he would want out of it. It was merely intended to be mentioned in passing to illustrate the potential this idea has gathered within hours of its conception.
Anything that goes against our vision statement, goals, or ideals of leveling the playing field will NOT be tolerated. To address the "it's happening whether you help or not", that was first brought up by zjz (in different terms) and we were all in agreement. Someone, somewhere, is going to do this regardless of WSB's support. Do we want it to be us, who we know have good intentions, or the other people?

Comment (TheWallStreetAnalytics) [reply ReactFrontEnd/README.md:4]: I felt like I should respond directly, but Matt has already posted what I was going to say. I do apologize for any confusion there, but I believe there was some miscommunication. We are not planning to sell out to an investor or anything nor get free work for a "bullshit startup".

Comment (dirjk) [reply ReactFrontEnd/README.md:4]: no thanks.

**Comment by android606** [comment]
Domain name wsbanalytics.org is available.

**Pull request closed without merging**